### ISSUE TITLE
Homography-Based Loss Function for Camera Pose Regression. (arXiv:2205.01937v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#18611]
Homography-Based Loss Function for Camera Pose Regression. (arXiv:2205.01937v1 [cs.CV])<br>
https://ift.tt/NZlbtqY<br>
<p>Some recent visual-based relocalization algorithms rely on deep learning methods to perform camera pose regression from image data. This paper focuses on the loss functions that embed the error between two poses to perform deep learning based camera pose regression. Existing loss functions are either difficult-to-tune multi-objective functions or present unstable reprojection errors that rely on ground truth 3D scene points and require a two-step training. To deal with these issues, we introduce a novel loss function which is based on a multiplane homography integration. This new function does not require prior initialization and only depends on physically interpretable hyperparameters. Furthermore, the experiments carried out on well established relocalization datasets show that it minimizes best the mean square reprojection error during training when compared with existing loss functions.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
